app_dart/test/src/service/fake_graphql_client.dart

```diff
@@ -38,20 +38,20 @@ class FakeGraphQLClient implements GraphQLClient {
   void verifyQueries(List<QueryOptions> expected) {
     expect(queries.length, expected.length);
     for (int i = 0; i < queries.length; i++) {
-      /// [BaseOptions.toKey] serializes all of the relevant parts of the query
-      /// or mutation for us, except the fetch policy.
-      expect(queries[i].toString(), expected[i].toString());
-      expect(queries[i].fetchPolicy, expected[i].fetchPolicy);
+      expect(
+        queries[i].properties,
+        equals(expected[i].properties),
+      );
     }
   }
 
   void verifyMutations(List<MutationOptions> expected) {
     expect(mutations.length, expected.length);
     for (int i = 0; i < mutations.length; i++) {
-      /// [BaseOptions.toKey] serializes all of the relevant parts of the query
-      /// or mutation for us, except the fetch policy.
-      expect(mutations[i].toString(), expected[i].toString());
-      expect(mutations[i].fetchPolicy, expected[i].fetchPolicy);
```

Comment:
This is not working with the current version of graphql library. The toString now returns `Instance of <MutationOptions>`. The list of `properties` includes `policies`, so we can leave only one `expect`